django/publicmapping/redistricting/templates/viewplan.html

```diff
@@ -1219,7 +1244,7 @@ <h3>{% blocktrans %}You can select up to three statistics at once to display in
         $('#steps').tabs({ show: function(event, ui) {
             if ($(this).tabs('option', 'selected') == 3) {
                 exportIndexFile.init();
-                createTwitterButton();
+                createShareButtons();
```

Comment:
When you resolve the merge conflict you have, you'll notice that 
```js
createTwitterButton();
```
was changed to 
```js
if (typeof createTwitterButton === 'function') {
  createTwitterButton();
}
```
This is because that function is `undefined` if the plan isn't yet shared and throws a console error. Your implementation is throwing the same error and I recommend resolving merge conflicts and using the change similarly.